### PR TITLE
Configuration clarity improvements

### PR DESCRIPTION
**Pull request by Litemage**
# What's In This PR?

Users previously had to sort through bit flags defined in the preprocessor to pass to `MAX31856::config`. Now, two new enums have been defined for clarity to users:

`Tc_Type` and `Max31856_Conversion_Mode`

This improves clarity for users, and also simplifies what users see using the auto-complete feature inside of the Arduino IDE

Also, there are now a couple of examples for thermocouple R3aktor boards that involves logging to the SD card and dumping measurements to the Serial port

### CODE REVIEW

File: src/PWFusion_MAX31856.h [hunk 1]
```diff
@@ -66,6 +66,22 @@ typedef enum Max31856_Reg_e {
   REG_SR      = 0x0F  // Status Register
 } Max31856_Reg;
 
+typedef enum {
```
Comment (CodingWithFusion) [inline src/PWFusion_MAX31856.h:69]: Would you add these new enum values to keywords.txt in the root folder of the project?   The 1.x Arduino IDE has pretty terrible syntax highlighting and requires you to list all the keywords for it to highlight.

**Review by CodingWithFusion — CHANGES_REQUESTED** [review]

File: src/PWFusion_MAX31856.h (hunk 1 above)
Comment (Litemage) [reply src/PWFusion_MAX31856.h:69]: Fixed in 1c0e667cec8ff3531e8d1644f32b13843cc2e90e

Comment (CodingWithFusion) [reply src/PWFusion_MAX31856.h:69]: This is stupid, but...   Can you try using the library with a 1.x IDE.   I think you have to have exactly one tab between keywords and values.    If it works as is, go ahead and merge this

Comment (Litemage) [reply src/PWFusion_MAX31856.h:69]: Nope, you're right, strait from Arduino CLI docs:

> keywords.txt is formatted in four fields which are separated by a single true tab (not spaces):

I'll get it fixed

Comment (Litemage) [reply src/PWFusion_MAX31856.h:69]: Converted to proper syntax in 19b5db4

**Pull request merged**